packages/ignite/src/templates/plugin/index.js.ejs

```diff
@@ -0,0 +1,54 @@
+// Ignite plugin for <%= props.name %>
+// ----------------------------------------------------------------------------
+
+const NPM_MODULE_NAME = 'react-native-MODULENAME'
+const EXAMPLE_FILE = '<%= props.name %>Example.js'
+const PLUGIN_PATH = __dirname
+const APP_PATH = process.cwd()
+
+const add = async function (context) {
+  const { ignite, filesystem } = context
+
+  // install a npm module and link it
+  await ignite.addModule(NPM_MODULE_NAME, { link: true })
+
+  // copy the component example file (if examples are turned on)
+  await ignite.addComponentExample(EXAMPLE_FILE, { title: '<%= props.name %> Example' })
+
+  // Example of copying templates/<%= props.name %> to App/<%= props.name %>
+  // if (!filesystem.exists(`${APP_PATH}/App/<%= props.name %>`)) {
+  //   filesystem.copy(`${PLUGIN_PATH}/templates/<%= props.name %>`, `${APP_PATH}/App/<%= props.name %>`)
```

Comment:
Probably -- I'd imagine this file will see the most modifications as we start using this generator. I'll leave that until later if that's cool.